README.md

```diff
@@ -818,16 +819,25 @@ Refer to the relevant Yaskawa Motoman documentation for more information on how
 
 After robot-calibration, the transform between the shared `world` frame and each robot's `base` frame will be known, and MotoROS2 will include it in the transforms it broadcasts.
 
-### Some group combinations won't publish data
+### Best-effort reliability topics fail to publish large messages
```

Comment:
I'd suggest using a heading that only describes the symptom, not a/the cause.

Users are more likely to search for symptoms they've observed.

The description below would also need to be updated to match -- or perhaps we keep the version that @ted-miller initially wrote.

The `cause of this behaviour is unknown` would be where this would go into details and link the issue that's linked in the *description* now.